ballerina-integrator/real-time-error-detection/Ballerina.toml

```
@@ -0,0 +1,4 @@
+[package]
+org = "gayaldassanayake"
+name = "real_time_error_detection"
```

Lets update the name to be consistent with naming of other samples
```suggestion
name = "real_time_error_notifier"
```

---

Resolved with dc174b3e08f4b01963858cb14c3cdc1dbb85e50f